### DISCUSSION TITLE
Verifier not supported Node.js backend issue using testnet

### DISCUSSION THREAD

**Discussion by alammcajnp** [Web3Auth/Web3Auth#735] [Help]
Hi Web3Auth Community, 

I'm using testnet, at Node js Backend. Error and web3auth verifier screenshot attached.  

My torus account stop working, while previously it was working at the backend.
Getting error message - 'Verifier not supported'.

Previously same verifier was working. 

Let me explain the process which I am following,
old verifier named (simubladedoughfiverifier (testnet)) was working, then suddenly I received an error (Verifier not supported), then I created a new verifier (torus-testnet-auth(on testnet, auth0 custom)) at web3auth but unfortunately, I am receiving same error (Verifier not supported), then I created second verifier (DouSim(on testnet, auth0 other)),  it also producing same error, then I created third one  (simublade-test-torus(on testnet, google)) it's also generated the same issue, 

Now I'm tired, please help me to solve out it.

![Screenshot from 2022-10-18 12-48-28](https://user-images.githubusercontent.com/6941054/196362737-b6f2ae38-1c0a-493b-b24c-e6a821f5a761.png)

![Screenshot from 2022-10-18 11-46-58](https://user-images.githubusercontent.com/6941054/196362787-d3f309bd-6ed3-4832-a878-503799267670.png)

**Comment by chaitanyapotti** [comment]
Please refer to the discussion here: https://github.com/orgs/Web3Auth/discussions/685

**Comment by alammcajnp** [comment]
@chaitanyapotti  checked above discussion, also implemented as suggested but it's not helped to solve the issue
